src/Tasks/Microsoft.Common.CurrentVersion.targets

```
@@ -63,8 +63,9 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <AvailablePlatforms Condition="'$(VisualStudioVersion)' != '' and '$(VisualStudioVersion)' > '10.0'">Any CPU,x86,x64</AvailablePlatforms>
   </PropertyGroup>
 
-  <!-- Import does NOT apply to .NETCore -->
+  <!-- Import depends on if it is .NETCore. Imports for .NETFramework is a superset of that for .NETCore-->
   <Import Project="Microsoft.NETFramework.props" Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' or '$(TargetFrameworkIdentifier)' == 'Silverlight'"/>
+  <Import Project="Microsoft.NET.props" Condition="'$(TargetFrameworkIdentifier)' != '.NETFramework' and '$(TargetFrameworkIdentifier)' != 'Silverlight'"/>
```

Hmm, should this be more restrictive? It'll get imported to C++ projects now, too. Maybe 
```suggestion
  <Import Project="Microsoft.NET.props" Condition="'$(TargetFrameworkIdentifier)' == '.NETStandard' or '$(TargetFrameworkIdentifier)' != '.NETCoreApp'"/>
```

or similar?

---

Should this be `<Import Project="Microsoft.NET.props" Condition="'$(TargetFrameworkIdentifier)' == '.NETStandard' or '$(TargetFrameworkIdentifier)' == '.NETCoreApp'"/> ?`

---

c++ don't have a $(TargetFrameworkIdentifier) ?

---

you were right that I missed the second `!=`/`==`. Edited above.